docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -559,17 +559,22 @@ Step 3 produces the following code values:
 .KS
    Symbol Length   Code
    ------ ------   ----
-   A       3        010
-   B       3        011
-   C       3        100
-   D       3        101
-   E       3        110
-   F       2         00
+   A       3       010
+   B       3       011
+   C       3       100
+   D       3       101
+   E       3       110
+   F       2       00
    G       4       1110
    H       4       1111
 .KE
 .fi
 
+The above prefix code contains codes that are parsed from left to right
+(with MSB on the left). All subsequent prefix codes in this
```

Comment:
"with MSB on the left"

I think this should be written the other way around: "with the first bit of the code in the MSB position" or, maybe even clearer: "with the first bit of the code on the left".

And below: "with the first bit of the code in the LSB position" (note that we use this phrase in section 1.5.1.) or "with the first bit of the code on the right".